docs/docs/agents/streaming.md

```diff
@@ -212,6 +212,10 @@ You can specify multiple streaming modes by passing stream mode as a list: `stre
         print("\n")
     ```
 
+## Disable streaming
```

Comment:
This section heading is odd in the `Streaming` section -- the purpose is not clear